marklogic-data-hub/src/test/resources/test-projects/all-module-types/modules/services/myResourceExtension.sjs

```diff
@@ -0,0 +1,53 @@
+/**
+  Copyright (c) 2020 MarkLogic Corporation
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+'use strict';
+const DataHub = require("/data-hub/5/datahub.sjs");
+const datahub = new DataHub();
+
+
+function get(context, params) {
```

Comment:
Same principle as above - we don't need anything real in here, just have a "get" function that does nothing. Otherwise, the reader may think that what's in here matters, but it doesn't.